### PR TITLE
kp-diff 0.5.1 (new formula)

### PR DESCRIPTION
**Pull request by aivanovski**
- [ X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/k/kp-diff.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+class KpDiff < Formula
+  desc "CLI utility to diff KeePass database"
+  homepage "https://github.com/aivanovski/kp-diff"
+  url "https://github.com/aivanovski/kp-diff/releases/download/0.5.1/kp-diff.jar"
+  sha256 "e0c7d7b1bf6895a7f5cfb976fb6a33034b0ac5554b1ce21a4db6ede70c4556df"
+  license "Apache-2.0"
+
+  depends_on "openjdk@11"
```
Comment (SMillerDev) [inline Formula/k/kp-diff.rb:8]: Why not the more modern jdk?

File: Formula/k/kp-diff.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  def install
+    libexec.install "kp-diff.jar"
+    bin.write_jar_script libexec/"kp-diff.jar", "kp-diff", java_version: "11"
+  end
+
+  test do
+    # Setup test files
+    test_data = [
```
Comment (SMillerDev) [inline Formula/k/kp-diff.rb:17]: Can we just use normal resource blocks?

File: Formula/k/kp-diff.rb (hunk 1 above)
Comment (aivanovski) [reply Formula/k/kp-diff.rb:8]: I'm actually not sure what version of jdk use here. But as I know, usually java application are bundled with specific jre, what guarantees that the application will work correctly.
This application was compiled with jdk 11 and it was tested with it, that's why jdk 11 was chosen.

File: Formula/k/kp-diff.rb (hunk 2 above)
Comment (aivanovski) [reply Formula/k/kp-diff.rb]: Fixed

File: Formula/k/kp-diff.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/k/kp-diff.rb:8]: Homebrew requires everything use the latest version of dependencies unless that version is known to be broken, so this should be:
```suggestion
  depends_on "openjdk"
```

**Comment by aivanovski** [comment]
@SMillerDev Could you please explain what is wrong with the 'Commit style' check?

File: Formula/k/kp-diff.rb (hunk 1 above)
Comment (aivanovski) [reply Formula/k/kp-diff.rb]: Fixed

**Comment by SMillerDev** [comment]
All changes should be in one commit named like the first one

**Comment by aivanovski** [comment]
```
    * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
  Error: 1 problem in 1 formula detected.
 ```

**Pull request closed without merging**